build/xtask/src/dist.rs

```diff
@@ -606,6 +606,13 @@ fn build_archive(cfg: &PackageConfig, image_name: &str) -> Result<()> {
     archive
         .copy(chip_dir.join("openocd.gdb"), debug_dir.join("openocd.gdb"))?;
 
+    // Copy `qemu.sh` into the archive if it exists;
```

Comment:
Is this for humility?

Reply:
Yeah, Humility will need to be able to pull this script out of the archive.

Reply:
Yea, the archive should include everything humility needs to run.